client/custom-element-api-mock/index.ts

```diff
@@ -79,8 +82,8 @@ class FakeCustomElement implements ICustomElement {
       cb(
         {
           config: (window as any).customElement.config || {},
-          disabled: false,
-          value: (window as any).customElement.value || null,
+          disabled: ((window.top as any).getDisabled() && (window.top as any).getDisabled()) || false,
```

Comment:
Why the method is Called twice? 

```suggestion
          disabled: (window.top as any).getDisabled(),
```
Or you meant this? 

```suggestion
          disabled: ((window.top as any).getDisabled && (window.top as any).getDisabled()) || false,
```

Reply:
Or with the check for windows.top (not sure if `|| false` at the end is required according to the method implementation)

```suggestion
          disabled: (window.top && (window.top as any).getDisabled && (window.top as any).getDisabled()) || false,
```

Reply:
remove extra parenthesis